pixie/parser.pxi

```diff
@@ -192,9 +192,9 @@
 
 (defmacro parser
   "(parser nm inherits & rules)
```

Comment:
Can you remove the `nm` argument from here? I think it's also a copy-paste leftover, only `defparser` seems to need that as an argument.